### PR TITLE
Improve terminology in listing 6.15, move out of sync import to 6.18

### PR DESCRIPTION
**Pull request by treygrainger**

### CODE REVIEW

File: chapters/ch06/3.spell-correction.ipynb [hunk 1]
```diff
@@ -428,6 +413,14 @@
        "      <td>1</td>\n",
        "    </tr>\n",
        "    <tr>\n",
+       "      <th>61</th>\n",
+       "      <td>laptopa</td>\n",
+       "      <td>laptops</td>\n",
+       "      <td>6</td>\n",
+       "      <td>5565</td>\n",
```
Comment (treygrainger) [inline chapters/ch06/3.spell-correction.ipynb:420]: This looks very different than what was there before (5565 vs 14119). I wonder why?

File: chapters/ch06/3.spell-correction.ipynb [hunk 2]
```diff
@@ -507,22 +500,29 @@
...
-       "61      laptopa     laptop                6              14119          1\n",
        "62        latop     laptop                5              14119          1\n",
        "136     toucpad   touchpad                6              11550          1\n",
        "137    touxhpad   touchpad                5              11550          1\n",
        "148     wirless   wireless                6              10060          1\n",
-       "127     tableta     tablet                6               8260          1\n",
        "8          cage       case                6               7541          1\n",
        "10         cape       case                5               7541          1\n",
        "30      gallaxy     galaxy                6               5839          1\n",
+       "61      laptopa    laptops                6               5565          1\n",
```
Comment (treygrainger) [inline chapters/ch06/3.spell-correction.ipynb:525]: This looks very different than what was there before (5565 vs 14119). 

Also lost "tableta" in the process...

I wonder why?

File: chapters/ch06/3.spell-correction.ipynb [hunk 3]
```diff
@@ -578,7 +579,7 @@
        "  <thead>\n",
        "    <tr style=\"text-align: right;\">\n",
        "      <th></th>\n",
-       "      <th>misspell</th>\n",
+       "      <th>misspelling</th>\n",
```
Comment (treygrainger) [inline chapters/ch06/3.spell-correction.ipynb:582]: The committed notebook before was clearly wrong. The code is supposed to output "misspelling", but the output for this column was "misspell", which I'm assuming means the notebook likely wasn't rerun after the code change was made.

I've updated the tables in the manuscript to say "misspelling" for the column header to match the code, assuming the code is correct.

File: chapters/ch06/3.spell-correction.ipynb [hunk 4]
```diff
@@ -746,23 +723,44 @@
...
       ],
       "text/plain": [
-       "        misspell   correction  misspell_counts  correction_counts  edit_dist\n",
+       "     misspelling   correction  misspell_counts  correction_counts  edit_dist\n",
        "168        ipad.         ipad                6               7749          1\n",
        "142   hp tochpad  hp touchpad                6               7144          1\n",
        "143   hp toucpad  hp touchpad                6               7144          1\n",
        "176    iphone s4    iphone 4s                5               4642          2\n",
        "179    iphone4 s    iphone 4s                5               4642          2\n",
-       "180   iphones 4s    iphone 4s                5               4642          1\n",
```
Comment (treygrainger) [inline chapters/ch06/3.spell-correction.ipynb:760]: Do you know why the output is so different vs. before? Was it a library upgrade or something?

File: chapters/ch06/3.spell-correction.ipynb (hunk 2 above)
Comment (dcrouch26) [reply chapters/ch06/3.spell-correction.ipynb:525]: I'm not sure why your results are changed in this PR, but the main I have locally matches main on remote.

File: chapters/ch06/3.spell-correction.ipynb (hunk 3 above)
Comment (dcrouch26) [reply chapters/ch06/3.spell-correction.ipynb:582]: The code is correct.

File: chapters/ch06/3.spell-correction.ipynb (hunk 2 above)
Comment (dcrouch26) [reply chapters/ch06/3.spell-correction.ipynb:525]: It seems like before the algorithm was determining that "laptopa" was a mispelling of "laptop", but now "laptopa" is a mispelling of "laptops", still investigating

**Pull request closed without merging**